app/src/main/java/cz/muni/fi/pv256/movio2/uco_410034/App.java

```diff
@@ -18,6 +20,11 @@
     public void onCreate() {
         super.onCreate();
 
+        if (LeakCanary.isInAnalyzerProcess(this)) {
```

Comment:
idealne by to malo byt len pre debug verziu, nechces aby ti to tvojim uzivatelom zhadzovalo appku, ked ty niekde leaknes pamat.

Reply:
to je moj nazor ^